src/wallet/wallet.go

```diff
@@ -427,6 +586,35 @@ func (w *Wallet) CreateAndSignTransaction(vld Validator, unspent blockdb.Unspent
 	return &txn, nil
 }
 
+func (w *Wallet) setEncrypted(encrypt bool) {
+	if encrypt {
+		w.Meta["encrypted"] = "true"
```

Comment:
why is this saved as string? use bool

Reply:
The `Meta` is `map[string][string]`, maybe should use `map[string]interface{}`

Reply:
Oh, i thought it was `map[string]interface{}`,
leave this then

Reply:
I changed it to `map[string]interface{}`, so that the meta can have data type.